modules/gapi/src/backends/common/gmetabackend.cpp

```diff
@@ -85,6 +85,19 @@ class GGraphMetaBackendImpl final: public cv::gapi::GBackend::Priv {
                          const std::vector<cv::gimpl::Data>&) const override {
         return EPtr{new GraphMetaExecutable(graph, nodes)};
     }
+
+    virtual bool controlsMerge() const override
```

Comment:
@dmatveev FYI

Reply:
> what is about adding of test code for new functionality?

Added test: `seq_id`, `seqNo`, `timestamp`.
Already testing: `parseSSD`, `networks`
Didn't add test for `parseYolo` since there is no suitable model on CI